### PR TITLE
re #577 fix for running the bash file in a path with spaces

### PR DESCRIPTION
**Pull request by francisdb**
this fixes the issue for mac/linux
have not tested or modified anything windows related

### CODE REVIEW

**Comment by muuki88** [comment]
LGTM. I'll see if I can run the windows tests

File: src/main/resources/com/typesafe/sbt/packager/archetypes/bash-template [hunk 1]
```diff
@@ -235,9 +235,9 @@ run() {
   fi
   
   if [ -n "$custom_mainclass" ]; then
-    mainclass="$custom_mainclass"
+    mainclass=("$custom_mainclass")
```
Comment (muuki88) [inline src/main/resources/com/typesafe/sbt/packager/archetypes/bash-template]: what do the round brackets achieve here?

Comment (francisdb) [inline src/main/resources/com/typesafe/sbt/packager/archetypes/bash-template]: Needs to be an array (see also declare -a and [@] stuff)

**Comment by muuki88** [comment]
@francisdb I'm checking now if the jar builds are generally broken on windows. On linux we seem to be fine, still I have a strange feeling about this. #586

**Comment by francisdb** [comment]
@muuki88 since I did not touch windows stuff, the LauncherJarPlugin generally still works on non-with-spaces folders. The bat file still needs a fix to be run on paths with spaces (which is not uncommon for windows these days)

But since the linux/macos issue is fixed this can already be merged

**Comment by muuki88** [comment]
okay. will do this after the weekend

**Pull request merged**